### PR TITLE
flags support

### PR DESCRIPTION
**Pull request by kolovsky**

### CODE REVIEW

File: shaders/shaders_filterflags2.glsl [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<script id="filterflags2_fShader" type="x-shader/x-fragment">
+   precision mediump float;
+
+    varying vec4 col;
+
+    void main() {
+        gl_FragColor = col;
+  }
+</script>
+
```
Comment (jezekjan) [inline shaders/shaders_filterflags2.glsl]: What the difference between filterflags2 filterflags ?

File: shaders/shaders_flags.glsl [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	uniform mat4 rasterMatrix;
+	uniform sampler2D filter;
+	uniform float numfilters;
+	uniform float mapMatrix;
+	varying vec4 col;
+
+	void main() {
+
+
+
```
Comment (jezekjan) [inline shaders/shaders_flags.glsl]: Minor thing, but let's avoid these big white spaces. Use just one line space when needed.

File: shaders/shaders_flags.glsl [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  			else if (fdata[0]>=  ( (numfilters ) / 256.) && numfilters != 0. ){
+  				// data are selected
+				col = vec4(val, 0. , 0., 1.);
+			} else {
+  				// data visible but not selected
+  				col = vec4(0., val , 0., 1.);
+  			}
+
+
+
```
Comment (jezekjan) [inline shaders/shaders_flags.glsl]: same here. Use one better just one blank line.

File: shaders/shaders_filterflags2.glsl (hunk 1 above)
Comment (kolovsky) [reply shaders/shaders_filterflags2.glsl]: filterflags2 render all flags (sub-dimensions) to one raster and filterflags reads these values and writes it to indexTexture. This two step must be separated.

File: shaders/shaders_flags.glsl (hunk 2 above)
Comment (kolovsky) [reply shaders/shaders_flags.glsl]: ok

File: shaders/shaders_flags.glsl (hunk 3 above)
Comment (kolovsky) [reply shaders/shaders_flags.glsl]: ok

**Pull request merged**